app/src/main/res/values/colors.xml

```diff
@@ -4,10 +4,10 @@
     <color name="colorIsCurrentUserNotSick">#4CAF50</color>
```

Comment:
update colors

Reply:
Hey @madim, why don't you put these comments as TODOs in the code? We might forget them here.

Reply:
I'm planning to fix them in this PR